hashicorp/waypoint
# ui: update icons to use the FlightIcon package

## Related Issue
#2392 

## Description
This PR intends to transfer the Waypoint Ui over from using the `Pds::Icon` components to the new `FlightIcon` ones.

## How to Test
1. Pull this branch down `git checkout ui/flight-icon-update`
2. `yarn` to download the new packages
3. `yarn start` to boot up UI with mock data
4. Click around to ensure all icons are looking as expected!


## Additional Notes
`flight-icons` currently doesn't have any animated icons, so our `loading` spinner is not present. There's [discussion](https://github.com/hashicorp/flight/issues/282) about adding it that's picked up in recent days, so I imagine this will be added soon but I'm not planning on holding this PR back any longer since it's just 1 icon left.

## Screenshots
### Deployments Page
<img width="1438" alt="Screen Shot 2021-12-08 at 3 40 06 PM" src="https://user-images.githubusercontent.com/60155296/145282835-47bf5bb3-5bb2-4a05-87ec-608c6be64b40.png">

### Auth Page
<img width="1440" alt="Screen Shot 2021-12-08 at 3 51 04 PM" src="https://user-images.githubusercontent.com/60155296/145282878-86b5209e-253a-40d1-860f-4437c87693e6.png">


_As of making this PR, the method for altering the colors of the icons in `Flight` requires altering the `use` element instead of `path`. This also seems to prevent us from changing the stroke width. Feedback on how we could change the stroke width would be helpful (since I'm pretty sure it was thinner before)_
